pkg/rancher-desktop/backend/containerClient/mobyClient.ts

```diff
@@ -276,6 +281,7 @@ export class MobyClient implements ContainerEngineClient {
           delete links[linkName];
           console.debug(`Created directory ${ dirName }`);
         }));
+        await stream.promises.finished(entry.resume());
```

Comment:
Should the `default` case also resume the ignored entry?

Reply:
Good catch, fixed; also fixed multiple symlinks pointing to the same file, so anchore now installs.